ramls/loan.json

```diff
@@ -93,6 +93,10 @@
       "description": "Indicates whether or not this loan had its due date modified by a recall on the loaned item",
       "type": "boolean"
     },
+    "isDcb": {
```

Comment:
With this change we are altering the corresponding interface, which means we should also increase its version in both RAML and ModuleDescriptor